docs/_content/community/bijeenkomsten/20190131-demo-sprint-8.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Demo Sprint 8"
+date: '31-01-2019'
+---
+
+Den Haag, 31 januari 2019
+
+## Presentatie
+
+To Do
+
+## Slides:
+
+* [Demo Spint 8.pdf](../bestanden/zds2-demo-sprint-8.pdf)
+
+## Feedback en vragen
+
+To Do
```

Comment:
- Vraag van Centric n.a.v. de roadmap: zou het niet beter zijn om de epic over autorisatie naar voren te halen, omdat dit een belangrijk onderwerp voor het inbouwen van de koppeling? Reactie: epics autorisatie is een sprint naar voren gehaald door deze te wisselen met historie/versiebeheer.